demo/blocks/hero/hero-1.tsx

```
@@ -1,78 +1,77 @@
+                            <p className="text-muted-foreground line-clamp-1 text-sm leading-none max-sm:text-xs">
+                                Trusted by teams and businesses
+                            </p>
+                        </div>
+                    </div>
+                </div>
+
+                <TiltCard wrapperClassName="order-1 lg:order-2" className="bg-foreground/5 rounded-md p-2">
+                    <img
+                        src="https://images.unsplash.com/photo-1674027392842-29f8354e236c?w=1000"
+                        className="h-80 w-full rounded-md object-cover sm:h-90 lg:h-100"
+                        alt="Hero Image"
```

Biome (1.9.4)

[error] Avoid the words "image", "picture", or "photo" in img element alt text.

Screen readers announce img elements as "images", so it is not necessary to redeclare this in alternative text.

(lint/a11y/noRedundantAlt)